docs/source/changelog.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## Unreleased breaking changes
```

Comment:
I believe this will require a rebase given that we made a release in August 2025, see https://github.com/jupyterhub/repo2docker/releases/tag/2025.08.0.